supplements/extra-life/lost-laboratory-of-kwalish.xml

```diff
@@ -100,4 +100,411 @@
+		   <p class="indent">•Fill the armor with air, allowing you to breathe normally in any environment for up to 1 hour (1 charge or 5 hit points).</p>
+		   <p class="indent">•Gain darkvision to a range of 60 feet for up to 1 hour (1 charge or 5 hit points).</p>
+		   <p class="indent"><em>Powered Armor Options</em></p>
+		   <p class="indent"><i>Depending on where and how it appears in the adventure, you might wish to modify the features of Kwalish’s legendary powered armor.</i></p>
+		   <p class="indent"><b><i>Automatic Defenses.</i></b> Unless Kwalish deactivates the suit’s automatic defenses, no one can approach the armor without setting those defenses off. Treat the armor as a shield guardian that has stored a magic missile spell cast using a 4th-level spell slot. When the armor is reduced to 0 hit points, its defenses are rendered inert and it can be safely approached.</p>
+		   <p class="indent"><b><i>Battle of Wills.</i></b> When donned by a new user, the armor deems itself superior and attempts to take possession of that user. The user must succeed on a DC 13 Charisma saving throw or be possessed by the armor. While possessed, the user is incapacitated and loses control of its body but retains its awareness. The armor uses the possessed user’s statistics (as adjusted by the armor), but doesn’t gain access to the user’s knowledge, features, or proficiencies.</p>
+		   <p class="indent">Freeing a creature trapped inside the armor first requires defeating the armor’s automatic defenses (as above). The trapped creature can also attempt a DC 20 Charisma saving throw each day at dawn. On a successful save, the armor no longer controls the creature and can be safely donned by that creature at any time.</p>
+		   <p class="indent"><b><i>Stasis.</i></b> Whenever a creature wearing the armor drops to 0 hit points, the armor places that creature into a state of stasis. While in this state, the creature is stable and does not make death saving throws, but the armor takes control of the creature (as above). Additionally, the armor attempts to assume the identity of the user, assuring their allies that nothing is amiss. Freeing the user first requires defeating the armor’s automatic defenses (as above). A creature in stasis does not make Charisma saving throws to break the armor’s control.</p>
+		   <p class="indent"><b><i>Alternative Power.</i></b> Powered armor originally required energy cells to fuel it, but was adapted by Kwalish to be fueled by the life energy of the creature wearing it. You might decide that the armor can also draw power from additional sources, or that energy cells can be recharged with the aid of a tinker, inventor, or artificer. It might also be possible for allies to connect to the armor through the use of magic that generates a conduit something like an astral silver cord. While so connected, a willing ally can give up hit points as a reaction to fuel the armor’s abilities.</p>
```

Comment:
```suggestion
			<p class="indent">While wearing this armor, you gain the following benefits:</p>
			<ul>
				<li>You have a +1 bonus to AC.</li>
				<li>Your Strength score is 18 (this has no effect if your Strength is already 18 or higher).</li>
				<li>You have advantage on death saving throws.</li>
			</ul>
			<p class="indent">The armor has further capabilities that can be powered either by energy cells or by your own life energy. You can use a bonus action to draw power from an energy cell or sacrifice hit points to gain one of the following benefits:</p>
		   	<ul>
				<li>Emit a force field to gain 2d6 + 5 temporary hit points (1 charge or 5 hit points).</li>
				<li>Activate boosters to gain a flying speed of 15 feet for 1 minute (1 charge or 5 hit points).</li>
				<li>Fire arm-mounted laser: <i>Ranged Weapon Attack:</i> +8 to hit, range 120 feet, one target. <i>Hit:</i> 2d6 radiant damage (1 charge or 5 hit points).</li>
				<li>Translate any writing you can see in any nonmagical language, to a total of one thousand words over 1 minute (1 charge or 5 hit points).</li>
				<li>Fill the armor with air, allowing you to breathe normally in any environment for up to 1 hour (1 charge or 5 hit points).</li>
				<li>Gain darkvision to a range of 60 feet for up to 1 hour (1 charge or 5 hit points).</li>
			</ul>
			<p class="indent">The armor can accept only one energy cell at a time. It is found with one energy cell attached, containing 2d10 charges.</p>
			<div class="sidebar">
				<h5>Powered Armor Options</h5>
				<p>Depending on where and how it appears in the adventure, you might wish to modify the features of Kwalish’s legendary <i>powered armor</i>.</p>
				<p class="indent"><b><i>Automatic Defenses.</i></b> Unless Kwalish deactivates the suit’s automatic defenses, no one can approach the armor without setting those defenses off. Treat the armor as a <b>shield guardian</b> that has stored a <i>magic missile</i> spell cast using a 4th-level spell slot. When the armor is reduced to 0 hit points, its defenses are rendered inert and it can be safely approached.</p>
				<p class="indent"><b><i>Battle of Wills.</i></b> When donned by a new user, the armor deems itself superior and attempts to take possession of that user. The user must succeed on a DC 13 Charisma saving throw or be possessed by the armor. While possessed, the user is incapacitated and loses control of its body but retains its awareness. The armor uses the possessed user’s statistics (as adjusted by the armor), but doesn’t gain access to the user’s knowledge, features, or proficiencies.</p>
				<p class="indent">Freeing a creature trapped inside the armor first requires defeating the armor’s automatic defenses (as above). The trapped creature can also attempt a DC 20 Charisma saving throw each day at dawn. On a successful save, the armor no longer controls the creature and can be safely donned by that creature at any time.</p>
				<p class="indent"><b><i>Stasis.</i></b> Whenever a creature wearing the armor drops to 0 hit points, the armor places that creature into a state of stasis. While in this state, the creature is stable and does not make death saving throws, but the armor takes control of the creature (as above). Additionally, the armor attempts to assume the identity of the user, assuring their allies that nothing is amiss. Freeing the user first requires defeating the armor’s automatic defenses (as above). A creature in stasis does not make Charisma saving throws to break the armor’s control.</p>
				<p class="indent"><i><b>Alternative Power.</b> Powered armor</i> originally required energy cells to fuel it, but was adapted by Kwalish to be fueled by the life energy of the creature wearing it. You might decide that the armor can also draw power from additional sources, or that energy cells can be recharged with the aid of a tinker, inventor, or artificer. It might also be possible for allies to connect to the armor through the use of magic that generates a conduit something like an astral silver cord. While so connected, a willing ally can give up hit points as a reaction to fuel the armor’s abilities.</p>
			</div>
```